CONTRIBUTORS.md

```diff
@@ -19,4 +19,5 @@
 We also acknowledge contributions from the following collaborators:
 
 + [Florian Funke](http://www3.in.tum.de/~funkef/)
-+ [Michael Seibold](http://www3.in.tum.de/~seibold/)
\ No newline at end of file
++ [Michael Seibold](http://www3.in.tum.de/~seibold/)
++ [Oracle](https://github.com/oracle) (Copyright (c) 2023, Oracle and/or its affiliates.)
```

Comment:
Later we should consider to automate this and/or add additional authors from the history.

Reply:
#390